### ISSUE TITLE
Rewrite `run_docker` in Python for Windows users

### ISSUE THREAD

**Issue by ryanking13** [pyodide/pyodide#2440]
`run_docker` is a simple shell script that helps people to easily set dev environments for Pyodide.

Though it works well for Linux and Mac users, it would be great to rewrite it with Python (or docker-compose, etc) so that Windows users can also use it easily.

## Known issues

Accessing the Windows filesystem inside WSL2 (on which Docker Desktop for windows is based) is extremely slow. We should notify that clearly so that Windows users don't waste their time.

**Comment by rth** [comment]
Thanks for the issue. Since users would need WSL2 to run anyway, they can run `run_docker` from the bash terminal in WSL2 directly, can't they?

**Comment by ryanking13** [comment]
> Thanks for the issue. Since users would need WSL2 to run anyway, they can run `run_docker` from the bash terminal in WSL2 directly, can't they?

Yeah, that's true for WSL2 users. But I guess not all Windows users use WSL2 directly.
